api/app/domain/analyzers/land_cover_change_analyzer.py

```diff
@@ -0,0 +1,159 @@
+                land_cover_zarr_uri=self.land_cover_zarr_uri,
+                pixel_area_zarr_uri=self.pixel_area_zarr_uri,
+            )
+            dd_df_futures = await self.compute_engine.gather(
+                self.compute_engine.map(analysis_partial, aoi_list, geojsons)
+            )
+            dfs = await self.compute_engine.gather(dd_df_futures)
+            combined_results_df = await self.compute_engine.compute(dd.concat(dfs))
+
+        combined_results_df = combined_results_df[
+            combined_results_df.land_cover_change_area__ha > 0
+        ]
+        analyzed_analysis = Analysis(
+            combined_results_df.to_dict(orient="list"),
+            analysis.metadata,
+            AnalysisStatus.saved,
+        )
+        await self.analysis_repository.store_analysis(
+            land_cover_change_analytics_in.thumbprint(), analyzed_analysis
+        )
+
+    def analyze_admin_areas(self, gadm_ids):
+        query = f"select * from '{self.admin_results_uri}' where aoi_id in {gadm_ids} and land_cover_class_start != land_cover_class_end"
+        df = duckdb.query(query).df()
+        df["aoi_type"] = "admin"
+
+        return df
+
+    @staticmethod
+    def analyze_area(aoi, geojson, land_cover_zarr_uri, pixel_area_zarr_uri):
+        umd_land_cover = read_zarr_clipped_to_geojson(land_cover_zarr_uri, geojson)
+        pixel_area = read_zarr_clipped_to_geojson(pixel_area_zarr_uri, geojson)
+
+        lc_data_2015 = umd_land_cover.band_data.sel(year=2015)
+        lc_data_2024 = umd_land_cover.band_data.sel(year=2024)
+
+        # Create unique identifiers for each transition using a row×column encoding:
+        # row=start_class (2015), column=end_class (2024)
+        # Each transition gets unique id = row*9 + column
+        # e.g. class 2->5 transition = 2*9 + 5 = 23
+        lc_class_change = (
+            lc_data_2015 * LandCoverChangeAnalyzer.number_of_classes + lc_data_2024
+        )
+        lc_class_change.name = "class_change"
+
+        land_cover_change = xarray_reduce(
+            pixel_area,
+            lc_class_change,
+            func="sum",
+            expected_groups=(np.arange(81),),
+        )
+
+        land_cover_change_df = LandCoverChangeAnalyzer._post_process_results(
+            land_cover_change
+        )
+        land_cover_change_df["aoi_type"] = aoi["type"].lower()
+        land_cover_change_df["aoi_id"] = (
+            aoi["id"] if "id" in aoi else aoi["properties"]["id"]
+        )
+
+        return land_cover_change_df
+
+    @staticmethod
+    def _post_process_results(result_ds):
+        land_cover_change_ddf = (
+            result_ds.to_dask_dataframe()
+            .drop("band", axis=1)
+            .drop("spatial_ref", axis=1)
+            .reset_index(drop=True)
+        )
+
+        land_cover_change_ddf["land_cover_class_start"] = (
+            land_cover_change_ddf.class_change.apply(
+                lambda x: LandCoverChangeAnalyzer.land_cover_mapping[x // 9]
```

Comment:
Above you use `LandCoverChangeAnalyzer.number_of_classes` instead of directly using 9, probably want to use that here too